app/models/incident.rb

```diff
@@ -120,44 +134,71 @@ def reviewed?
     staff_reviews.present?
   end
 
-  # from https://gist.github.com/janko-m/2b2cea3e8e21d9232fb9
-  def to_claims_sql(table:)
+  # Fields we can't provide are commented where they appear in the claims schema.
+  # Field we're not providing in the first stage are implemented but commented.
+  def claims_fields(table:)
     report = driver_incident_report
 
     fields = {
       incident: {
+        # FilePrefix, FileNum
+        'DateEntered' => Date.today.strftime('%Y-%m-%d'),
+        # AppraisalMade, AppraisalNote
+        'IncidentDate' => report.occurred_at,
         street: report.location,
         city: report.town,
         state: 'MA',
-        # zip: driver.incident_report.zip,
+        zip: report.zip,
         longitude: longitude,
         latitude: latitude,
-        'IncidentDesc' => root_cause_analysis,
-        'VehicleRouteNum' => report.block,
-        'VehicleNum' => 777, # will be pulled from claims table
-        'PointOfContact' => report.damage_to_bus_point_of_impact,
-        'Status' => 'ir',
-        reason1: reason_code.identifier,
-        reason2: second_reason_code.try(:first, 3),
+        # AccidentCode, AccidentCodeGroup
         'Company' => driver.division.claims_id,
-        'DateEntered' => Date.today.strftime('%Y-%m-%d'),
-        'IncidentDate' => report.occurred_at,
+        'IncidentDesc' => root_cause_analysis,
         'EmployeeID' => driver.badge_number,
-        'Driver' => 999, # will be pulled from claims table
+        'Driver' => 999, # TODO: pull from claims table
         'DriverDesc' => report.description,
+        'VehicleRouteNum' => report.route,
+        # VehicleDestination
+        'VehicleNum' => 777, # TODO: pull from claims table
```

Comment:
This will be implemented properly (along with a matching AR model) once I have access to that database and can see the table details.